src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
+const HEALTH = 100;
+const DAMAGE = 50;
```

Comment:
```suggestion
const DEFAULT_HEALTH = 100;
const HERBIVORES_DAMAGE = 50;
```
The variables names should be iinformative